### DISCUSSION TITLE
RoborockZoneCleaningCapability could use Math.max and Math.min

### DISCUSSION THREAD

**Discussion by Flole998** [Hypfer/Valetudo#1514] [Archive]
RIght now at in the code there is this:
https://github.com/Hypfer/Valetudo/blob/fe65df3696d2a0af608db27c54f32b49bcb88635/backend/lib/robots/roborock/capabilities/RoborockZoneCleaningCapability.js#L25
Essentially this (and the following lines) are just own implementations of Math.max and Math.min. For better readability I suggest changing those to Math.max and Math.min.

**Comment by Hypfer** [comment]
Indeed. Thanks!

560a50edfca23c6a80378f0bed9cc3c1a533d8bc
